tests/e2e/rosa_autoscaler_test.go

```diff
@@ -42,15 +43,17 @@ var _ = Describe("Autoscaler", labels.Feature.Autoscaler, func() {
 			labels.Critical, labels.Runtime.Day2,
 			func() {
 				By("Retrieve help for create/list/describe/delete autoscaler")
-				_, err := rosaClient.AutoScaler.CreateAutoScaler(clusterID, "-h")
+				_, err := rosaClient.AutoScaler.CreateAutoScaler("-c", clusterID, "-h")
 				Expect(err).ToNot(HaveOccurred())
-				_, err = rosaClient.AutoScaler.EditAutoScaler(clusterID, "-h")
+				_, err = rosaClient.AutoScaler.EditAutoScaler("-c", clusterID, "-h")
```

Comment:
`-c` should be integrated to the command itself like others

Reply:
But I have to test a scenario, where I can't set cluster id

Reply:
@xueli181114 Changed as you said Thanks